### PR TITLE
✨ Introduce initial get wallets endpoints

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Add two endpoints to the api server:
image
image

## Related issue/s
- https://github.com/ITISFoundation/osparc-simcore/issues/4768

### CODE REVIEW

File: services/api-server/src/simcore_service_api_server/core/settings.py [hunk 1]
```diff
@@ -33,6 +33,15 @@ class WebServerSettings(BaseCustomSettings, MixinServiceSettings, MixinSessionSe
     )
     WEBSERVER_SESSION_NAME: str = "osparc.WEBAPI_SESSION"
 
+    @cached_property
+    def base_url(self) -> str:
+        # http://webserver:8080/v0
```
Comment (pcrespov) [inline services/api-server/src/simcore_service_api_server/core/settings.py:38]: I guess this comment should be `https://webserver:8080/`

File: services/api-server/tests/unit/conftest.py [hunk 2]
```diff
@@ -258,12 +258,10 @@ def mocked_webserver_service_api_base(
 
     # pylint: disable=not-context-manager
     with respx.mock(
-        base_url=settings.API_SERVER_WEBSERVER.api_base_url,
+        base_url=settings.API_SERVER_WEBSERVER.base_url,
         assert_all_called=False,
         assert_all_mocked=True,
     ) as respx_mock:
-        # WARNING: For this service, DO NOT include /v0 in the `path` to match !!!!
```
Comment (pcrespov) [inline services/api-server/tests/unit/conftest.py:265]: thx. I should have done this long time ago. Now is uniform everywhere

**Review by pcrespov — APPROVED** [review]
🎉 thx!

NOTE: Wallets depend on the product, not only the user. For the moment the api-server knows nothing about products. I still have to introduce them in the login mechanism (the api-secret+key should be associated to a product) . It is one of the tasks in https://github.com/ITISFoundation/osparc-simcore/issues/4752

File: services/api-server/src/simcore_service_api_server/core/settings.py (hunk 1 above)
Comment (bisgaard-itis) [reply services/api-server/src/simcore_service_api_server/core/settings.py:38]: Yep, good point. Corrected it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -14,6 +14,7 @@
   ⚠️    Changes in ops configuration etc. are required before deploying.
         [ Please add a link to the associated ops-issue or PR, such as in https://github.com/ITISFoundation/osparc-ops-environments or https://git.speag.com/oSparc/osparc-infra ]
   🗃️    Database table changed (relevant for devops).
+  👽️    Public API changes (meaning: dev features are moved to being exposed in production)
```
Comment (bisgaard-itis) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: @pcrespov @matusdrobuliak66 FYI I add here an icon to indicate changes in API server

File: services/api-server/tests/mocks/get_job_wallet_found.json
Comment (GitHK) [inline services/api-server/tests/mocks/get_job_wallet_found.json:1]: A bit out of place, no API server version bump?

**Review by GitHK — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (matusdrobuliak66) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Haha, approved!

**Review by matusdrobuliak66 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (matusdrobuliak66) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: probably add it to the title of the issue :+1:

Comment (bisgaard-itis) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: I added `include_in_schema=API_SERVER_DEV_FEATURES_ENABLED` to each of the endpoints, which means they are still dev features. I.e. they will be visible on the master deployment, but not staging and production. I will add the icon once this is removed and the features are exposed to the public (or the 👽️s)

**Pull request merged**